tests/output/ConsoleInteractionTest.py

```diff
@@ -712,9 +712,7 @@ def test_show_bear_with_content(self):
                              '  Needed Settings:\n'
                              '   * setting1: Required Setting.\n\n'
                              '  Optional Settings:\n'
-                             '   * setting2: Optional Setting. ('
-                             "Optional, defaults to 'None'."
-                             ')\n\n'
+                             '   * setting2: Optional Setting.\n\n'
```

Comment:
actually I was just thinking, why do we need that extra tuple with 'Optional'... since we are working with ```optional_params``` it is implicit that they are "Optional"

Or if you want to keep this 'Optional' then maybe we can just display ```<param_name> (defaults to 'something')``` instead of ```<param_name> (Optional, defaults to 'something')``` because we are displaying this info about params under the heading "Optional Parameters"... so it doesn't make sense to display "Optional" every time...